packages/create-gatsby/package.json

```diff
@@ -41,5 +41,8 @@
     "url": "https://github.com/gatsbyjs/gatsby.git",
     "directory": "packages/create-gatsby"
   },
-  "author": "Matt Kane <matt@gatsbyjs.com>"
+  "author": "Matt Kane <matt@gatsbyjs.com>",
+  "dependencies": {
```

Comment:
Please add everything to devDependencies, because we compile it all into a bundle